WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint

```diff
@@ -16,23 +16,18 @@
  history_interval                    = 60, 60,                               ! history output file interval in minutes (integer only)
  frames_per_outfile                  = 1, 1,                                 ! number of output times bulked into each history file
  restart                             = __ISRESTART__,                        ! whether this run is a restart
- restart_interval                    = 1440,                                 ! restart output file interval in minutes
+ restart_interval                    = 360,                                 ! restart output file interval in minutes
```

Comment:
The restart_interval should be greater than 360, i.e. we have used 1440. Please, could you change this back?